detox/src/android/espressoapi/UIDevice.js

```diff
@@ -151,7 +151,7 @@ class UiDevice {
   }
 
   static pressKeyCode(element, keyCode) {
-    function pressKeyCode1(keyCode) {
+    function pressKeyCode2(element, keyCode) {
```

Comment:
This is the result of the new generator invocation. I've temporarily enabled and then redisabled the generation of uidevice.js (see changes in `generation/index.js`)

Reply:
Does that mean that the new implementation is incorrect to some extent?

Reply:
What I meant was that I've regenerated UIDevice locally after the changes and then had it removed from generation's files list